.github/workflows/publish-images.yaml

```diff
@@ -0,0 +1,57 @@
+name: Publish Container Images
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  publish-container-images:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v2
+      - name: Login to Quay.io
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.QUAY_USERNAME }}
```

Comment:
I've already configured a Quay robot account and set its credentials as secrets in this repository

Reply:
cool deal